tests/testenv/cli_test_env.go

```diff
@@ -77,7 +77,7 @@ func NewCLITest(t *testing.T) *CLITest {
 		t.Name(),
 		"/", "_"), "\\", "_"), ":", "_")
 
-	logsDir := filepath.Join(os.TempDir(), "kopia-logs", cleanName+"."+clock.Now().Local().Format("20060102150405"))
+	logsDir := filepath.Join(t.TempDir(), "kopia-logs", cleanName+"."+clock.Now().Local().Format("20060102150405"))
```

Comment:
This seems like a spurious change.
Probably it'd be rejected upstream. The change affects many other tests besides the robustness ones.

IIRC, the intention was to be able to preserve the logs for debugging failed tests. I believe by default `t.TempDir()` is deleted even if the test fails.

Reply:
Sure, let me fix that.